04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+\begin{enumerate}
+    \item Changed the dependency to be based on the input resources rather than the number of islands participating. This added a more realistic element to the dilemma as in reality a hunt’s return would be based on the amount of resources (people, food, water and materials) entered rather than the number of islands participating.
+    \item Instead of a specified return we have a probabilistic return which adds a randomness element to the game to prevent pattern recognition. These probabilities can be tied to the population, disasters, forecasting and number of animals hunted.  
+\end{enumerate}
+
+\newpage
+\subsection{Functions within Foraging}
+\subsubsection{Inputs, outputs and Parameters}
+
+This is an element implemented in the code base. It determines the method of foraging, which is either deer hunting being payoff focused or fishing being risk aversion.\\
+
+
+\subsubsection{Primary Code Function}
+
+\begin{table}[h]
+\begin{center}
+\begin{tabular}{|p{1.5in}|p{1.5in}|p{1.5in}|} \hline
+\textbf{Functions} & \textbf{Inputs} & \textbf{Outputs} \\ \hline
+\texttt{TotalInput} & -Island Number\newline -Foraging amount & -Total foraging amount \\ \hline
+\texttt{deerUtilityTier} & -Total foraging amount\newline -Maximum deer per hunt\newline -Decay value & -Tier (according to input resources) \\ \hline
+\texttt{deerReturn} & -Bernoulli's probability\newline -Exponential decay & -Weight of the deer (0 if deer was not caught) \\ \hline
+\texttt{Hunt} & -Tier\newline -Weight of deer & -Utility (cumulative sum of the weight of the deers) \\ \hline
+\end{tabular}
+\caption{\label{tab:table-name}Foraging Deer Hunt Method's main functions}
+\end{center}
+\end{table}
+
+\subsubsection{Tier System}
+
+The deer hunting and fishing is divided into tiers (the number of fish that can be caught in a given day) which depends on the total input resources. Where the “zero” tier represents the cost of getting to the hunting location.\\ 
+
+Beyond the zero tier we have the cost of catching another deer/fish, which decreases as it is easier to catch the second animal than the first since we have already arrived at the hunting location. It should be noted there is a limit to how many animals we can catch in a given day.\\
+
+The focus of fishing is to avoid risk at the expense of payoff. Thus all the tier requirements are lower than the deers however the return is also lower, in addition to this the “zero” tier has been removed. Thus, if only 1 island goes fishing they are still expected to have a low return as long as they have enough resources to catch the first fish. \\
+
+The focus of deer hunting is higher payoffs therefore it has better returns but all of its tier thresholds are significantly higher and it also has a “zero” tier. Therefore it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.
```

Comment:
break the sentence??
comma after 'therefore' is suggested.

overall: bad sentence -- revise.

Reply:
Variety in connective words would make the writing flow more easily - thus is in this writing a lot and there are a lot of other ways to link sentences - don't be afraid to add variety :)) 

Suggested rewrite:
The focus of fishing is to avoid risk at the expense of payoff. This means that all tier requirements are lower than the deers. However, the return is also lower and in addition, the “zero” tier is also removed. Thus, if only 1 island goes fishing they are still expected to have a low return as long as they have enough resources to catch the first fish. \\

// it is the benefit of deer hunting not the focus
The benefit of deer hunting is higher payoffs. However, these improved returns come at the cost of significantly higher tier thresholds and a “zero” tier. Therefore, it is possible for a single island to go deer hunting without enough resources to reach the hunting location leading to no return.